config/tcs_config.toml

```diff
@@ -13,10 +13,8 @@
 # limitations under the License.
 
 [KvStorage]
-StoragePath = "config/Kv_Shared_tmp"
-StorageSize = "1 TB"
 # the remote version is of higher priority if enabled
-#remote_url = "http://localhost:9090"
+remote_url = "http://avalon-lmdb:9090"
```

Comment:
This will no longer work in standalone mode without Docker.
You should leave `#remote_url = "http://localhost:9090"` commented out and add a comment that one is for Docker and the other is for standalone.

You also need to mention in the "Standalone Build" section of `BUILD.md` to uncomment the `remote_url = "http://localhost:9090"` line and comment out the avalon-lmdb line.